Assets/Scripts/Augment/ProjectileController.cs

```diff
@@ -76,6 +76,15 @@ public class ProjectileController : MonoBehaviour
 
     public float lifeTime = 2f;
 
+    // Transfer modifier functions to clones
```

Comment:
Now I finally understand the timing headaches I had with the previous implementation